src/main/java/org/takes/http/BkBasic.java

```diff
@@ -87,16 +110,44 @@ public void accept(final Socket socket) throws IOException {
     }
 
     /**
+     * Whether to http connection is persistent or not.
+     * @param req RqLive
+     * @return Boolean
+     * @throws IOException if fails
+     */
+    private boolean persistent(final RqLive req) throws IOException {
+        boolean keep = false;
+        final RqHeaders.Base rqHeaders = new RqHeaders.Base(req);
+        if (rqHeaders.head().iterator().hasNext()) {
+            final Iterator<String> values = rqHeaders.header(BkBasic.CONNECTION)
+                .iterator();
+            if (values.hasNext()) {
+                do {
+                    keep = BkBasic.KEEP_ALIVE.toLowerCase(
+                        Locale.ENGLISH
+                        ).equals(
+                        values.next().toLowerCase(Locale.ENGLISH)
```

Comment:
@mstzn36 `RqHeaders` returns headers already in lower case, no need to do this conversion here again